cluster/src/main/java/io/scalecube/cluster/gossip/GossipProtocolImpl.java

```diff
@@ -58,15 +60,14 @@
   private List<Member> remoteMembers = new ArrayList<>();
   private int remoteMembersIndex = -1;
 
-  // Subscriptions
+  // Disposables
 
-  private Subscriber<Member> onMemberAddedEventSubscriber;
-  private Subscriber<Member> onMemberRemovedEventSubscriber;
-  private Subscriber<Message> onGossipRequestSubscriber;
+  private final Disposable.Composite actionsDisposables = Disposables.composite();
 
   // Subject
 
-  private Subject<Message, Message> subject = PublishSubject.<Message>create().toSerialized();
+  private FluxProcessor<Message, Message> subject = DirectProcessor.<Message>create().serialize();
```

Comment:
Why is here `.serialize()` but at [line](https://github.com/scalecube/scalecube-cluster/pull/19/files#diff-68fbb221b4dd9257bc5980ae7df47b8bR63)  it's simple `.create()` ? Isn't it should be similar everywhere ?

Reply:
Yeah, I guess it should be everywhere, but since the elements' supplying goes only through the `FluxSink` then it will be only the matter of style

Reply:
@artem-v what line do you refer to?

Reply:
Now it's great